pkg/controller/pkg/revision/reconciler.go

```diff
@@ -155,18 +156,26 @@ func WithLinter(l parser.Linter) ReconcilerOption {
 	}
 }
 
+// WithVersioner specifies how the Reconciler should versionerally parse a package.
```

Comment:
Is "versionerally" a real word? 🤔

Reply:
Oh - I think this is probably from `s/generic/version` haha.

Reply:
I don't believe it is a word... but I like it haha.. cleaning up 👍